server/sonar-server/src/main/java/org/sonar/server/permission/ProjectId.java

```diff
@@ -36,10 +36,12 @@
 
   private final long id;
   private final String uuid;
+  private final boolean isPrivate;
```

Comment:
`private` is no **ProjectId**entifier. We should either rename the class or not have this field here.